plugin/core/rpc.py

```diff
@@ -12,9 +12,9 @@
 DEFAULT_SYNC_REQUEST_TIMEOUT = 1.0
 
 
-def format_request(payload: Dict[str, Any]) -> str:
+def json_format_request(payload: Dict[str, Any], compact=True) -> str:
```

Comment:
Renamed it for clarity and also as it had the same name as one method in this file, making it confusing when searching for that name across the project.